lambdas/service/app.py

```diff
@@ -1538,6 +1542,17 @@ def handle_manifest_generation_request() -> Tuple[int, Manifest]:
                                                            catalog=catalog,
                                                            filters=filters)
         if manifest is not None:
+            url = furl(app.self_url())
+            if 'fetch' in url.path.segments:
+                url.path.segments = [seg for seg in url.path.segments if seg != 'fetch']
```

Comment:
Could just use `list.remove` here for the record, but not if you extract this to a method.